spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfiguration.java

```diff
@@ -114,9 +118,11 @@ public EurekaClientConfigBean eurekaClientConfigBean() {
 	@Bean
 	@ConditionalOnMissingBean(value = EurekaInstanceConfig.class, search = SearchStrategy.CURRENT)
 	public EurekaInstanceConfigBean eurekaInstanceConfigBean(InetUtils inetUtils) {
-		RelaxedPropertyResolver relaxedPropertyResolver = new RelaxedPropertyResolver(env, "eureka.instance.");
+		RelaxedPropertyResolver relaxedPropertyResolver = new RelaxedPropertyResolver(env,
```

Comment:
That one is is undesired, my bad. BTW thanks for the filtered classpath trick, I'll look into it.